src/validation_workflow/api_test_cases.py

```diff
@@ -0,0 +1,51 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import time
+from typing import Any
+
+from validation_workflow.api_request import ApiTest
+from validation_workflow.validation_args import ValidationArgs
+
+'''
+This class is the collection of test cases to run.
+'''
+
+
+class ApiTestCases:
+
+    def __init__(self) -> None:
+        self.OS_image_version = ValidationArgs().stg_tag('OS').split(' ')[0]
+        self.OSD_image_version = ValidationArgs().stg_tag('OSD').split(' ')[0]
```

Comment:
Is there any reason to use OS/OSD instead of opensearch/dashboards? OS can be very confused with `os` packages, or similar, and this is not a good practice to use acronyms as var names.a

Reply:
Thanks, @peterzhuamazon, we will use the fullname to be more clear. Was trying to avoid using long argument name.